packaging/debian/rules

```diff
@@ -22,19 +22,28 @@ override_dh_auto_install:
 	install -p -m 0644 instance_configs.cfg debian/google-guest-agent/usr/share/google-guest-agent
 	install -d debian/google-guest-agent/lib/systemd/system
 	install -p -m 0644 gce-workload-cert-refresh.timer debian/google-guest-agent/lib/systemd/system/
+	if [[ -d google-guest-agent ]]; then\
+		install -p -m 0644 google-guest-agent/cmd/google_guest_agent/google_guest_agent debian/google-guest-agent/usr/bin/google_guest_agent_manager;\
+	fi
 
 override_dh_golang:
 	# We don't use any packaged dependencies, so skip dh_golang step.
 
 override_dh_auto_build:
 	dh_auto_build -O--buildsystem=golang -- -ldflags="-s -w -X main.version=$(VERSION)" -mod=readonly
+	if [[ -d google-guest-agent ]]; then\
+		make -C google-guest-agent cmd/google_guest_agent/google_guest_agent;\
+	fi
 
 override_dh_installinit:
 	# We don't ship sysvinit files or need script changes for them.
 
 override_dh_systemd_enable:
 	install -d debian/google-guest-agent/lib/systemd/system
 	install -p -m 0644 *.service debian/google-guest-agent/lib/systemd/system/
+	if [[ -d google-guest-agent ]]; then\
+		install -p -m 0644 google-guest-agent/build/configs/usr/lib/systemd/system/google-guest-agent.service debian/google-guest-agent/lib/systemd/system/google-guest-agent-manager.service;\
```

Comment:
the plugin manager `.service` file should be [google-guest-agent-manager.service from legacy guest agent code base](https://github.com/GoogleCloudPlatform/guest-agent/blob/main/google-guest-agent-manager.service).

Reply:
The `.service` file should still be in the directory above since `google-guest-agent` is cloned into the `guest-agent` codebase, which becomes the root of the build staging directory.

Reply:
my point is that you are installing the wrong `.service` file, you are installing the one from new guest agent when you should be installing the one within legacy's code base.

Reply:
The legacy codebase's `.service` files should be covered by the line above,